docs/tech-reference/yum-plugins.rst

```diff
@@ -620,11 +620,11 @@ configuration values are optional.
  package signature verification.
 
 ``recursive`` Boolean flag. If true, units are copied together with their
-"latest" dependencies. False by default.
+"latest" dependencies. False by default. More information about its usage can be found in :ref:`advanced_copy_between_repositories`.
```

Comment:
It might be clearer if written as "If true, units are copied together with the latest versions of their dependencies".